pdp/attribute.go

```diff
@@ -89,13 +89,13 @@ func (v AttributeValueType) describe() string {
 	case DataTypeSetOfNetworks:
 		items := []string{}
 		i := 0
-		for _, n := range v.Value.([]net.IPNet) {
+		for n := range v.Value.(*SetOfNetworks).Iterate() {
```

Comment:
What happens if v.Value has the wrong type? Shouldn't you be using the x, ok := type assertion idiom here?